.cspell/project-words.txt

```diff
@@ -23,6 +23,7 @@ miri
 monitorable
 mpoint
 NVML
+pagetables
```

Comment:
hot take: `pagetables`, `mpoint`, and others should not be in the dictionary
I am now wondering whether it's a good idea to spell-check every string...

Reply:
(this does not need to be resolved now)